recipes/gen3/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "4.26.1" %}
+
+package:
+  name: gen3
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/g/gen3/gen3-{{ version }}.tar.gz
+  sha256: 4572b67b499c7660d41fdb0acac6fa10bdf0e3953528710914ce6c7341b14091
+
+build:
+  entry_points:
+    - gen3 = gen3.cli.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python ={{ python_min }}
+    - poetry >=1.4.1
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }}
+    - requests
+    - indexclient >=2.3.0,<3.0.0
+    - drsclient >=0.3.0
+    - aiohttp
+    - backoff
+    - cdislogging >=1.1.0,<2.0.0
+    - click
+    - fhirclient
+    - jsonschema
+    - dataclasses-json <=0.5.9
+    # - pypfb >=0.5.30
+    - tqdm >=4.61.2,<5.0.0
+    - humanfriendly
+    - python-dateutil
+    - aiofiles >=0.8.0,<0.9.0
+    - pandas >=1.4.2
+    - urllib3 >2.0.0
+    - httpx
+    - xmltodict >=0.13.0,<0.14.0
+    - pyyaml >=6.0.1
+    - gen3users
+
+test:
+  imports:
+    - gen3
+  commands:
+    # - pip check
```

Comment:
Why? If it is something upstream this package, we should try to fix it.

Reply:
This will fail because of a circular dependency on `pypfb`, which itself depends on `gen3`.  The only solution I could come up with so far is to leave these commented out and re-enable later.  The `gen3` library itself can be imported without `pypfb`, but the CLI interface tries to import, and `pip check` will complain that `pypfb` isn't installed.  I'm open to other solutions but this was the only thing that worked for me so far.